### ISSUE TITLE
Encrypt pre-signatures and triples

### ISSUE THREAD

**Issue by volovyks** [sig-net/mpc#54]
### Description
We must encrypt pre-signatures and tripels before they are stored in the persistent DB to avoid risks in case a malicious party gets access to them.

**Comment by ChaoticTempest** [comment]
This is pretty much a non-issue as the DB resides on the node, which we also have the private share as well. So getting access to the node would also mean getting access to the private share. Encrypting the presignature and triples would give us no extra advantage here

**Comment by volovyks** [comment]
That is true, but we have discussed this and agreed it can be an additional level of security.
And Redis will live separately in Kube, not in the node.
